db/interface.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding:utf-8 -*-
+
+from db.models import Payment, Detail
+
+
+class PaymentService(object):
+    """
+    Serves as an interface which takes requests from the front-end
+    and runs the corresponding actions in the database.
+    """
+
+    def __init__(self):
```

Comment:
The new branch looks good, although I would prefer to keep the db stuff in its own directory.
You may have had import issues before because there was a circular-like import chain happening.